src/Microsoft.Fx.Portability.MetadataReader/Microsoft.Fx.Portability.MetadataReader.project.json

```diff
@@ -1,18 +1,18 @@
 ﻿{
   "supports": {
-    "net46.app": { },
-    "dnxcore50.app": { }
+    "net46.app": { }
   },
   "dependencies": {
     "MicroBuild.Core": "0.2.0",
     "Microsoft.NETCore": "5.0.0",
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-beta-23516",
-    "System.Reflection.Metadata": "1.2.0-rc2-23713",
-    "System.Security.Cryptography.Algorithms": "4.0.0-beta-23328"
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.0",
```

Comment:
Curious, why the downgrade for `Microsoft.NETCore.Portability.Compatibility`?
[Getting Started: Update your packages to the latest](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/getting-started-core.md#update-your-packages-to-the-latest)